examples/fakegroundstation/ground_station_service.py

```diff
@@ -1,3 +1,6 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Sorry, I am not very clear about my comment. Could you add copy rights text to each code?

```
# Copyright 2019 Infostellar, Inc.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#    http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

Reply:
Huh, I guess I completely missed that part in printing_client.py. Not sure how that happened